app/views/backoffice/audit/index.en.html.erb

```diff
@@ -1,27 +1,25 @@
 <% title 'Back office: audit records' %>
 
-<div class="grid-row">
-  <div class="column-full">
-    <h1 class="heading-xlarge gv-u-heading-xxlarge">Back office: audit records</h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-xl">Back office: audit records</h1>
 
-    <p class="heading-medium">
-      Actions performed in the last 30 days
-    </p>
+    <h3 class="govuk-heading-m"> Actions performed in the last 30 days</h3>
 
-    <table class="backoffice-table">
-      <thead>
-        <tr>
-          <th>Timestamp</th>
-          <th>Action</th>
-          <th>Author</th>
+    <table class="govuk-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header">Timestamp</th>
```

Comment:
We can reduce the column widths using `govuk-!-width-one-third` (as the idea is to leave the 3rd column more space as it will contain full names and emails that are longer, so it doesn't wrap too much)

You can see before we were using a hardcoded width in the <td>